java/core/src/test/java/com/whylogs/core/metrics/TestMetric.java

```diff
@@ -0,0 +1,23 @@
+package com.whylogs.core.metrics;
+
+import java.util.ArrayList;
+import org.junit.Assert;
+import org.testng.annotations.Test;
+
+@Test
+public class TestMetric {
+
+  @Test
+  public void testMetrics() {
+    ArrayList<Metric> metrics = new ArrayList<>();
+    metrics.add(IntegralMetric.zero(new MetricConfig()));
+    metrics.add(IntegralMetric.zero(new MetricConfig()));
+
+    for (Metric metric : metrics) {
+      Assert.assertTrue(metric instanceof IntegralMetric);
+      metric.merge(new IntegralMetric());
```

Comment:
Did you want to assert the `Metric` returned by the merge is an `IntegralMetric`?